scripts/vagrant/zeppelin-dev/roles/maven/files/apache-mirror-selector.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python
```

Comment:
possibly license here - the original author doesn't seem to specify licensing term...